ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -17,7 +17,38 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 // See the feature message definitions for descriptions.
 message ExperimentalFeatures {
   ExperimentalSelfServiceErrorCodes self_service_error_codes = 1;
+  CommandDeduplicationFeatures command_deduplication = 2;
 }
 
 // GRPC self-service error codes are returned by the Ledger API.
 message ExperimentalSelfServiceErrorCodes {}
+
+
+message CommandDeduplicationFeatures {
```

Comment:
Consider adding a comment that provides context as to the reason for these features, their stability, and their target state.